### ISSUE TITLE
[Feature request] Optional GPS log for some counters

### ISSUE THREAD

**Issue by Eloitor** [albertvaka/bettercounter#57]
In addition to the time of the counters, I also need to keep track of the GPS location of some counters...

**Comment by albertvaka** [comment]
I don't think this is something I will implement. It's the first time someone requests it and it would require a lot of effort to get it done: dealing with the GPS API, modifying the format of the data (including the exported files, somehow keeping backwards compatibility), probably adding a way to visualize the locations (otherwise it's not that useful), allowing a setting to enable/disable this (some people might not like it, or just don't need it an want to save battery), etc.
